ext/spl/spl_directory.c

```diff
@@ -2310,6 +2310,10 @@ PHP_METHOD(SplFileObject, fgetcsv)
 		if (esc_len == 0) {
 			escape = PHP_CSV_NO_ESCAPE;
 		} else {
+			php_error_docref(NULL, E_DEPRECATED, "Passing a non-empty string to the $escape parameter is deprecated since 8.4");
```

Comment:
I think that in the past we didn't explicitly specify the version of the deprecation; if this is the preferred style now, I'm fine with it (although, according the doc style guides, "as of 8.4.0" might be preferable).

Reply:
see https://wiki.php.net/rfc/deprecated_attribute and #14750

Reply:
Ah, thank you. Then this is fine.